src/test/storage/storage_manager_test.cpp

```diff
@@ -50,4 +51,34 @@ TEST_F(StorageStorageManagerTest, HasTable) {
   EXPECT_EQ(storage_manager.has_table("first_table"), true);
 }
 
+TEST_F(StorageStorageManagerTest, TableNames) {
+  auto& storage_manager = StorageManager::get();
+  auto table_names = storage_manager.table_names();
+  // The insertion order is different to return value order due to hashing
```

Comment:
The actual hash functions are implementation-dependent, and the unordered map does not guarantee an order. Thus, this test case might break with a different hash implementation.